request.go

```diff
@@ -1926,6 +1926,62 @@ func (r *Request) WithMultipart() *Request {
 	return r
 }
 
+//	req := NewRequestC(config, "GET", "/path")
+//	req.WithClient(&http.Client{
+//	  Transport: &http.Transport{
+//		DisableCompression: true,
+//	  },
+//	})
+
+// WithReporter sets reporter.
+//
+// The new reporter overwrites AssertionHandler.Reporter. It will be used to
+// report formatted fatal failure messages
```

Comment:
Not quite true. It overwrites the whole AssertionHandler, not AssertionHandler.Reporter. And the new AssertionHandler is DefaulyAssertionHandler with specified reporter + existing formatter + nil Logger. Could you please mention all this in comment?